tests/kuttl/common/scripts/check_cluster_status.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# List of Northbound pods
+pods=("ovsdbserver-nb-0" "ovsdbserver-nb-1" "ovsdbserver-nb-2")
```

Comment:
This seems unnecessarily hardcoded. I suggest you make the script accept an argument that would specify the number of expected pods, (and the type of the database.)